scripts/gitlab/test.sh

```diff
@@ -19,7 +19,7 @@ git submodule update --init --recursive
 rustup show
 if [[ "${RUST_FILES_MODIFIED}" == "0" ]];
 then echo "__________Skipping Rust tests since no Rust files modified__________";
-else ./test.sh || exit $?;
+else ./test.sh --no-release || exit $?;
```

Comment:
I thought we were running test with `--release` to speed things up, also some tests may time out like this `has been running for over 60 seconds`?

Reply:
It would be nice to see some (compile + run time) numbers and to check if
```rust
[profile.dev]
opt-level = 1
```
is faster for tests.

Reply:
> It would be nice to see some (compile + run time) numbers

Yes that what I was doing last night but fell asleep eventually :laughing: 

Stay tuned for numbers. You are probably right with the timeouts.